api/solarperformanceinsight_api/models.py

```diff
@@ -168,6 +174,7 @@ class PVsystModuleParameters(BaseModel):
         ),
         gt=0,
     )
+    _pvlib_dc_model: str = PrivateAttr("pvsyst")
```

Comment:
might want to name these attributes `_modelchain_dc_model` to avoid confusion with the pvlib function names.

Reply:
and similar for the other `_pvlib*` attributes